firestore/src/android/wrapper.cc

```diff
@@ -56,11 +59,12 @@ Wrapper::Wrapper(Wrapper* rhs) : Wrapper() {
 
 Wrapper::~Wrapper() = default;
 
-jni::Env Wrapper::GetEnv() const { return firestore_->GetEnv(); }
-
-Object Wrapper::ToJava(const FieldValue& value) {
-  return FieldValueInternal::ToJava(value);
+jni::Local<jni::Object> Wrapper::ToJava() const {
```

Comment:
Remove the `jni::` qualifiers. This was required in the .h file, but the `using` declarations near the top of this file mean they're not needed here. Add new `using` declarations if you have to.